index.html

```diff
@@ -19,7 +19,7 @@
                 &lt;script src='stream.min.js'&gt;&lt;/script&gt;
             </pre>
 
-            <a class='download' href="stream.min.js">Download stream.js</a>
+            <a class='download' href="stream.js.zip">Download stream.js</a>
```

Comment:
Instead of hotlinking to us (github), it might be more beneficial to get stream.js into [cdnjs](https://github.com/cdnjs/cdnjs#readme) or [jsDelivr](https://www.jsdelivr.com/).